src/content-helper/editor-sidebar/related-posts/related-posts.scss

```diff
@@ -0,0 +1,246 @@
+@import "../../common/css/variables";
+@import "../../common/css/functions";
+
+.wp-parsely-related-posts {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+
+	.related-posts-description {
+		display: flex;
+		padding: to_rem(6px) 0 var(--grid-unit-20, to_rem(16px)) 0;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: to_rem(10px);
+		align-self: stretch;
+
+		/* Helper Text */
+		font-size: to_rem(13px);
+		font-style: normal;
+		font-weight: 400;
+		line-height: to_rem(20px); /* 153.846% */
+	}
+
+	.related-posts-body {
+		display: flex;
+		flex-direction: column;
+		padding: to_rem(6px) 0 var(--grid-unit-20, to_rem(16px)) 0;
+		align-items: flex-start;
+		gap: to_rem(16px);
+		align-self: stretch;
+
+		> div {
+			width: 100%;
+		}
+
+		.related-posts-settings {
+			width: 100%;
+			display: flex;
+			flex-direction: column;
+			gap: to_rem(16px);
+			align-self: stretch;
+
+			.components-base-control__field {
+				margin-bottom: 0;
+			}
+
+			.components-select-control {
+				display: flex;
+				align-items: flex-start;
+				gap: to_rem(16px);
+				align-self: stretch;
+				height: to_rem(36px);
+
+				.components-input-control__container {
+					padding: to_rem(3px);
+				}
+			}
+
+			.components-input-control__prefix {
+				margin-left: to_rem(8px);
+			}
+
+			.components-input-control {
+				display: flex;
+				height: to_rem(36px);
+				padding: var(--grid-unit-15, to_rem(12px)) var(--grid-unit-10, to_rem(8px)) var(--grid-unit-15, to_rem(12px)) var(--grid-unit-20, to_rem(16px));
+				align-items: center;
+				gap: var(--grid-unit-5, to_rem(4px));
+				align-self: stretch;
+			}
+		}
+
+		.related-posts-filter-settings {
+			display: flex;
+			flex-direction: column;
+			gap: var(--grid-unit-20, to_rem(16px));
+			width: 100%;
+
+			.related-posts-filter-types {
+				width: 100%;
+
+				.components-toggle-group-control {
+					height: to_rem(40px);
+
+					/**
+					 * Workaround for the ToggleGroupControl animation:
+					 * Prevents the animation from exiting the container.
+					 */
+					overflow: hidden;
+
+					/**
+					 * Workaround animation bug with ToggleGroupControl.
+					 * This works by setting the button background to transparent and
+					 * then setting the active item background to the sidebar black color.
+					 * The background should only pop-up as visible after 0.5s, therefore
+					 * hiding any janky animation.
+					 */
+					button {
+						background: transparent;
+						outline: transparent solid 2px;
+						outline-offset: -3px;
+
+						&[data-active-item] {
+							background: var(--sidebar-black, #1e1e1e);
+							transition:
+								background 0s 0.5s,
+								border-radius 0s 0.5s,
+								box-shadow 0s 0.5s;
+							box-shadow: 0 -3px 0 0 #fff, 0 3px 0 0 #fff;
+							border-radius: 3px;
```

Comment:
Missing `to_rem`s in 5 instances here, unless we don't want to do that for 2px or 3px.

Reply:
I believe we cannot use rem units for box-shadow, outlines and border-radius